prime-router/src/test/kotlin/fhirengine/azure/FHIRTranslatorIntegrationTests.kt

```diff
@@ -191,6 +196,20 @@ class FHIRTranslatorIntegrationTests : Logging {
             QueueAccess.sendMessage(any(), any())
         }
 
+        // check events
+        assertThat(azureEventService.reportStreamEvents[ReportStreamEventName.ITEM_TRANSFORMED]!!).hasSize(1)
+        assertThat(
+            azureEventService
+                .reportStreamEvents[ReportStreamEventName.ITEM_TRANSFORMED]!!.first()
+        ).isInstanceOf<ReportStreamItemEvent>()
+        val event = azureEventService
+            .reportStreamEvents[ReportStreamEventName.ITEM_TRANSFORMED]!!.first() as ReportStreamItemEvent
+        assertThat(event.params[ReportStreamEventProperties.ORIGINAL_FORMAT]).isEqualTo("FHIR")
+        assertThat(event.params[ReportStreamEventProperties.TARGET_FORMAT]).isEqualTo("HL7")
+        assertThat(event.params[ReportStreamEventProperties.RECEIVER_NAME]).isEqualTo("phd.x")
+        val enrichments = event.params[ReportStreamEventProperties.ENRICHMENTS] as List<*>
```

Comment:
nit: should be able to verify the actual value here, not just that it exists?

Reply:
De-nit'ed with latest code push.